code/modules/clothing/head/helmet.dm

```diff
@@ -171,7 +171,8 @@
 	var/helmet_overlays[]
 	flags_inventory = BLOCKSHARPOBJ
 	flags_inv_hide = HIDEEARS
-	var/flags_marine_helmet = HELMET_SQUAD_OVERLAY|HELMET_GARB_OVERLAY|HELMET_DAMAGE_OVERLAY|HELMET_STORE_GARB
+	var/flags_marine_helmet = HELMET_SQUAD_OVERLAY|HELMET_GARB_OVERLAY|HELMET_DAMAGE_OVERLAY|HELMET_STORE_GARB|HELMET_VOX_ALTERNATIVE
+	var/vox_icon_state = "helmet_vox"
```

Comment:
make it a subtype instead

Reply:
I'll see what I can do, I did it this way because that way I can make it so normal helmets work on Vox without issue.

Reply:
we should probably implement a feature that makes vendors recognize and offer different stock for other races

Reply:
I'll work on it, vendor code is awful from what I briefly read.